llvm/lib/CAS/OnDiskCAS.cpp

```diff
@@ -57,15 +57,11 @@ class TrieRecord {
   };
 
   enum class ObjectKind : uint8_t {
-    /// Node: refs and data.
+    /// Object: refs and data.
     Invalid = 0,
 
-    /// Node: refs and data.
-    Node = 1,
-
-    /// Tree: custom node. Pairs of refs pointing at target (arbitrary object)
-    /// and names (String), and some data to describe the kind of the entry.
-    Tree = 3,
+    /// Object: refs and data.
+    Object = 1,
 
     /// String: data, no alignment guarantee, null-terminated.
     String = 4,
```

Comment:
Does String still get used?

Reply:
There is no interface to request it but the implementation is not removed yet. It could be useful if we want to bring back the internal blob objects that doesn't require a hash.

Maybe remove this in a separate PR (there are few other things can be removed at the same time) so it is easier to bring back if needed?